src/pages/ReplaceServer/PrepareController.js

```diff
@@ -0,0 +1,180 @@
+    }
+  }
+
+  renderPlaybookProgress () {
+    const steps = [
+      {
+        label: translate('deploy.progress.commit'),
+        playbooks: ['commit']
+      },
+      {
+        label: translate('deploy.progress.config-processor-run'),
+        playbooks: ['config-processor-run.yml']
+      },
+      {
+        label: translate('deploy.progress.ready-deployment'),
+        playbooks: ['ready-deployment.yml']
+      },
+      {
+        label: translate('server.deploy.progress.rm-cobbler'),
+        playbooks: ['rm-cobbler']
+      },
+      {
+        label: translate('server.deploy.progress.rm-known-host'),
+        playbooks: ['known-hosts']
+      },
+      {
+        label: translate('server.deploy.progress.cobbler-deploy'),
+        playbooks: ['cobbler-deploy.yml']
+        // TODO:
+        //    cobbler-deploy *might* prompt for a password, although the logs on the QE system
+        //       ardana@10.84.81.17 indicate that none was supplied and it succesfully ran. Hmmmm
+      },
+    ];
+
+    const playbooks = [
+      {
+        name: 'commit',
+        action: ((logger) => {
+          const commitMessage = {'message': 'Committed via Ardana Installer'};
+          return postJson('/api/v1/clm/model/commit', commitMessage)
+            .then((response) => {
+              logger('Model committed\n');
+            })
+            .catch((error) => {
+              const message = translate('update.commit.failure', error.toString());
+              logger(message+'\n');
+              throw new Error(message);
+            });
+        }),
+      },
+      {
+        name: 'config-processor-run',
+      },
+      {
+        name: 'ready-deployment',
+      },
+      {
+        name: 'rm-cobbler',
+        action: ((logger) => {
+          return fetchJson('/api/v1/clm/cobbler/servers')
+            .then((response) => {
+              const cobbler_server = response.find((e) =>
+                e.ip === this.props.operationProps.server.ip ||
+                e.name === this.props.operationProps.server.name);
+              if (cobbler_server) {
+                const name = cobbler_server.name;
+
+                return deleteJson('/api/v1/clm/cobbler/servers/' + name)
+                  .then((response) => {
+                    logger('Host removed from cobbler\n');
+                  })
+                  .catch((error) => {
+                    const message = translate('update.remove_cobbler.failure', error.toString());
+                    logger(message+'\n');
+                    throw new Error(message);
+                  });
+
+              } else {
+                logger('Host not present in cobbler, continuing\n');
+              }
+            });
+        }),
+      },
+      {
+        name: 'known-hosts',
+        action: ((logger) => {
+          return deleteJson('/api/v1/clm/known_hosts/' + this.props.operationProps.server.id)
+            .then((response) => {
```

Comment:
I checked on my provo system, 
day0-cp1-c1-m1-mgmt ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBDSEpFr/J2HPBsUjAlfWzYwivCEGhh24I417PKCaCeg0OcdznKWBJYxp21mECJNkGgdaxb4b5Io/TAn2wVG+i28=
hostname is day0-cp1-c1-m1-mgmt.  this.props.operationProps.server.id is from servers.yml which in this case is controller1. so you can not use this.props.operationProps.server.id here But you can use server id + ip find it from  '/api/v1/clm/model/cp_internal/CloudModel.yaml'  cloudModel['internal']['servers']; in sever['hostname'] which should have day0-cp1-c1-m1-mgmt. 

 Off the topic: if you need set --limit down the road , need to use day0-cp1-c1-m1 which is in  server['ardana_ansible_host'].